site.yml

```diff
@@ -21,460 +21,460 @@ Documents:
         Slug: how-to-install-golang
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Installing Golang
+        SEO Description: The guide covers the basics of installing Golang
+        SEO Keywords: Golang
 
   - Minio Deployment:
       - Minio Deployment Quickstart Guide:
         Link: https://raw.githubusercontent.com/minio/minio/master/docs/orchestration/README.md
         Slug: minio-deployment-quickstart-guide
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | MinIO Deployment Quickstart Guide
+        SEO Description: This guide covers deployment documentation for Orchestration platforms - Docker Swarm, Docker Compose, Kubernetes, DC/OS
+        SEO Keywords: Docker Swarm, Docker Compose, Kubernetes, DC/OS
 
       - Deploy Minio on Docker Swarm:
         Link: https://raw.githubusercontent.com/minio/minio/master/docs/orchestration/docker-swarm/README.md
         Slug: deploy-minio-on-docker-swarm
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Deploy MinIO on Docker Swarm
+        SEO Description: This guide covers how MinIO server can be easily deployed in distributed mode on Swarm to create a multi-tenant, highly-available and scalable object store.
+        SEO Keywords: Docker Swarm, Docker Compose
 
       - Deploy Minio on Kubernetes:
         Link: https://raw.githubusercontent.com/minio/minio/master/docs/orchestration/kubernetes/README.md
         Slug: deploy-minio-on-kubernetes
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Deploy MinIO on Kubernetes
+        SEO Description: This guide documents the multiple options to deploy MinIO on Kubernetes. Concepts like Deployments and StatefulSets MinIO in standalone, distributed or gateway mode.
+        SEO Keywords: kubernetes, statefulsets, helm chart repository
 
       - Deploy Minio on DC/OS:
         Link: https://raw.githubusercontent.com/minio/minio/master/docs/orchestration/dcos/README.md
         Slug: deploy-minio-on-dc-os
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Deploy MinIO on DC/OS
+        SEO Description: This guide documents the deployment of MinIO on DC/OS
+        SEO Keywords: DC/OS, Marathon-LB
 
       - Deploy Minio on Docker Compose:
         Link: https://raw.githubusercontent.com/minio/minio/master/docs/orchestration/docker-compose/README.md
         Slug: deploy-minio-on-docker-compose
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | Deploy MinIO on Docker Compose
+        SEO Description: This guide covers deployment of MinIO using Docker Compose. Compose allows defining and running single host, multi-container Docker applications.
+        SEO Keywords: Docker compose, distributed minio, single host, multi-container
```

Comment:
I think we can add keywords indicating that this is documentation to deploy MinIO on Docker Compose